src/backend/db/products.js

```diff
@@ -112,28 +112,172 @@ export const products = [
   },
   {
     _id: "58027fcf-1854-449b-9607-27ceefe049d1",
-    title: "Bolton Planters",
+    title: "Mini Couple Cups",
     price: {
-      discounted: 4999,
-      original: 5999
+      discounted: 199,
+      original: 999
     },
     rating:4,
     imgURL:comboSet2,
-    categoryName: "Gifts",
+    categoryName: "Cups",
     offer:"combo-offer",
+    isBestseller: false,
+  },
+  {
+    _id: "8s7079e3-5467-4012-9d58-596887022259",
```

Comment:
instead of that you can use uuid

Reply:
I had used `uuid` before. But a new uuid is generated on page reload. 
Hence, to create single product page feature, I had to change it to fixed uuid.